studio/components/interfaces/Organization/BillingSettingsV2/TaxID/TaxID.tsx

```diff
@@ -128,8 +128,9 @@ const TaxID = () => {
         <div className="sticky space-y-6 top-16">
           <div>
             <p className="text-base">Tax ID</p>
-            <p className="text-sm text-scale-1000">
-              If you would like to include specific tax ID(s) to your invoices.
+            <p className="text-sm text-scale-1000 pr-4">
+              If you would like to include specific tax ID(s) to your invoices. Old invoices are not
```

Comment:
just to keep the same language as payment methods
```suggestion
              If you would like to include specific tax ID(s) to your invoices. Past invoices are not
```